src/Mod/Mesh/App/Exporter.h

```diff
@@ -42,42 +43,25 @@ namespace Mesh
  * Constructors of derived classes are expected to be required, for passing
  * in the name of output file.
  *
- * Objects are added using the addMeshFeat(), addPartFeat(), etc.
- *
  * If objects are meant to be combined into a single file, then the file should
  * be saved from the derived class' destructor.
  */
 class Exporter
 {
     public:
         Exporter();
+        virtual ~Exporter() = default;
 
-        /*!
-         * \return true if \a is an object that can be exported as mesh.
-         */
-        static bool isSupported(App::DocumentObject *obj);
-
-        virtual bool addMeshFeat(App::DocumentObject *obj) = 0;
-        virtual bool addPartFeat(App::DocumentObject *obj, float tol) = 0;
-
-        /// Recursively adds objects from App::Part & App::DocumentObjectGroup
-        /*!
-         * \return true if all applicable objects within the group were
-         * added successfully.
-         */
-        bool addAppGroup(App::DocumentObject *obj, float tol);
-
-        bool addObject(App::DocumentObject *obj, float tol);
+        int addObject(App::DocumentObject *obj, float tol);
```

Comment:
```suggestion
        /// Add object and all subobjects and links etc. Returns the number of stuff added.
        /*!
         * @param tol The tolerance/accuracy with which to generate the triangle mesh
         * @return The number of objects/subobjects that was exported from the document.
                   See the parameter `accuracy` of ComplexGeoData::getFaces
         */
        int addObject(App::DocumentObject *obj, float tol);
```

Reply:
Added new commits for the suggested documentation